src/T2DMap.h

```diff
@@ -41,6 +41,8 @@
 #include <QtConcurrent>
 #include "post_guard.h"
 
+#include <QList>
```

Comment:
This line should be shifted up before the `#include "post_guard.h"` header file...